website/blog/2023-12-14-release-1.6.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Release Notes - Podman Desktop 1.6
+description: Podman Desktop 1.6 has been released!
+slug: podman-desktop-release-1.6
+authors: slemeur
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, compose, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.6/SantaSeal.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.6 Release! 🎉
+
+![Podman-desktop-1-6-hero](img/podman-desktop-release-1.6/SantaSeal.png)
+
+This release introduces:
+
+- **Minikube featured extension**: Minikube extension to create local Kubernetes clusters in containers.
+- **Podman 4.8.2**: [Podman 4.8.2](https://github.com/containers/podman/releases) is now included in Windows and Mac installers.
+- **Setting page for Command-Line tools**: Manage and update your CLI tools.
+- **Kubernetes contexts Manager**: Browse all your kubernetes contexts, set default and remove unused ones.
+- **Editable Podman Machine for MacOS**: Easy resize and reconfiguration of the Podman runtime environment.
+- **Filters for Containers and Pods lists**: Focus on the containers and Pods you are working with.
+- **Sorting on Volumes and Images list**: Sort volumes or images with your prefered criterias.
+- **Enhancement to show environment colums on Containers and Pods lists**: Easy catch of the environment on which a container or a pod is running on.
+- **Extension API improvements**: Another set of improvements to the extension API enabling more goodness for 🦭 Podman Desktop's extensions
```

Comment:
All the above points have odd capitalization. I believe they should either have title capitalization (ex. Minikube Featured Extension), or just Minikube featured extension throughout the list.

There is no Vale error though, so either way is fine, WDYT @themr0c ?